src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--1">
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--2">
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--3">
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--4">
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--5">
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
+        <div class="stars__star stars__active"></div>
```

Comment:
there should be only one class here `stars__start`. the bg should be controlled with the modifier of the parent (`stars--#number`)